environment.ubuntu.yml

```diff
@@ -1,14 +1,73 @@
 name: cea
```

Comment:
What's the difference between this and the one used to build the docker image?

Reply:
not much difference at all! actually, they should be more or less the same.